examples/CMakeLists.txt

```diff
@@ -9,4 +9,16 @@ add_custom_target(install_libs
 add_subdirectory(bsoncxx)
 add_subdirectory(mongocxx)
 
-add_custom_target(examples DEPENDS ${MONGOCXX_EXAMPLES} ${BSONCXX_EXAMPLES})
+add_custom_target(examples DEPENDS ${MONGOCXX_EXAMPLE_EXECUTABLES} ${BSONCXX_EXAMPLE_EXECUTABLES})
+
+# Run all mongocxx examples on `make examples`.
```

Comment:
To run the tests, it's `make test`. Maybe `make example` is what we want? I guess "test" is a common verb, while "example" as a verb is less colloquial. Is "test" being used as a verb in "make test"?? How do you read make commands? o.o